weather/module.py

```diff
@@ -18,6 +21,93 @@ class Weather(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
+    def _get_useful_data(self, json, ctx, lang_preference):
+        """
+        example json: https://wttr.in/praha?lang=sk&format=j1
+        get useful data from json as list of individual days
+        """
+        weather = []
+        # get individual days to extract data
+        num_of_forecast_days = 3 # number of days to get forecast for (including current day, max is 3)
```

Comment:
I think this should be constant defined right after imports.

Reply:
Yeah, that is probably right. Move it before the class and use uppercase, as it will be constant.